src/main/java/com/google/devtools/build/lib/runtime/StarlarkOptionsParser.java

```diff
@@ -51,6 +55,10 @@
  */
 public class StarlarkOptionsParser {
 
+  // Use the plain strings rather than reaching into the Alias class and adding a dependency edge.
```

Comment:
I actually think this [DAMP](https://stackoverflow.com/questions/6453235/what-does-damp-not-dry-mean-when-talking-about-unit-tests) approach is underrated and underutilized in our code base. Clean text searching is useful and you can negate the added fragility with good test coverage (which should exist anyway).  

Is there any reason for even these private constants?